src/vanilla/Model/CodeModelJs.cs

```diff
@@ -230,9 +230,10 @@ public bool ContainsTimeSpan
         public string ConstructServiceClientJSExports()
         {
             StringBuilder builder = new StringBuilder();
-            builder.AppendLine($"exports.{Name} = {Name};");
-            builder.AppendLine($"exports.{GetServiceModelsName(Name)} = models;");
-            builder.AppendLine($"exports[\"default\"] = {Name};");
+            builder.AppendLine($"module.exports = {Name};");
+            builder.AppendLine($"module.exports['default'] = {Name};");

```

Comment:
@daschult - Out of curiosity: Was there a reason to use module.exports['default'] and not module.exports.default ?

Reply:
That was similar to the code that Typescript generated in some of my handwritten tests, so I went with that. I don't think there was any other reason.

Reply:
Maybe it's because default is a keyword in es6, so the expression `obj.default` isn't allowed.